lib/parser.js

```diff
@@ -496,4 +514,8 @@ function calculateNewBaseUrl(filePath, root, lookUp) {
   return calculate(filePath, []);
 }
 
+function calculateBoilerplateFilePath(targetFilePath) {
+  return path.resolve(path.join(process.cwd(), BOILERPLATE_DIRECTORY_NAME), path.basename(targetFilePath));
+}
+
```

Comment:
Move this above `calculateNewBaseUrl` (alphabetical order).

Reply:
> ```diff
> - const file = dependency.target;
> + const targetFilePath = dependency.target;
> const actualFilePath = dependency.actualFilePath;
>
>  return markbinder.includeFile(actualFilePath, {
>    baseUrlMap: this.baseUrlMap,
>    rootPath: this.rootPath,
> -   targetFilePath: file,
> +   targetFilePath: targetFilePath
>  })
> ```

can actually do

```diff
 return markbinder.includeFile(actualFilePath, {
   baseUrlMap: this.baseUrlMap,
    rootPath: this.rootPath,
-   targetFilePath: file,
+   targetFilePath
  })
```

Reply:
@nicholaschuayunzhi You can move your comment [here](https://github.com/MarkBind/markbind/pull/130#discussion_r165357626) :)

Reply:
I used `path.basename(targetFilePath)`. This means that it won't support having sub folders within the `_boilerplates` folder.